src/lib/objects/payggy/index.ts

```diff
@@ -15,7 +16,19 @@ export const payggyDescriptor: WakuObjectDescriptor = {
 
 	standalone: StandaloneComponent,
 
-	onMessage: (address, store, message) => {
+	onMessage: (address, adapter, store, message) => {
+		if (message?.data) {
+			const res = MessageDataSendSchema.safeParse(message.data)
+			if (res.success) {
+				const token = {
+					...res.data.token,
+					name: res.data.token.symbol,
+					amount: BigInt(0), // the amount is not really necessary for checkBalance
```

Comment:
maybe we can have a `Token` type that does not contain `amount` field?

Reply:
Good idea, I opened an issue about this: #177 

I would not include that in this PR.